command/format.go

```diff
@@ -385,21 +385,32 @@ func (t TableFormatter) OutputSecret(ui cli.Ui, secret *api.Secret) error {
 	}
 
 	if secret.Auth != nil {
```

Comment:
Just a minor comment, otherwise looks great!
Would you think we could avoid at least one level of nesting logic? like combining `secret.Auth != nil` with `secret.Auth.MFARequirement != nil` into a single if statement, then fixing the else part?

Reply:
I tried this out and I think I prefer the current formatting - I think it logically flows better, otherwise we have two separate code paths where `Auth != nil`, which doesn't feel as nice to me. I'll lean towards keeping it as-is unless there's a strong inclination to change, I think.